docs/language/expressions.rst

```diff
@@ -177,10 +178,10 @@ an interface at runtime, which can be used to write a `supportsInterface()` func
 in the EIP.
 
 The contract code for a contract, i.e. the binary WebAssembly or BPF, can be retrieved using the
-``type(c).creationCode`` and ``type(c).runtimeCode`` fields, as ``bytes``. In Ethereum,
+``type(c).creationCode`` and ``type(c).runtimeCode`` fields, as ``bytes``. On evm,
 the constructor code is in the ``creationCode`` WebAssembly and all the functions are in
```

Comment:
Why `creationCode` WebAssembly is the constructor for EVM, if we does not uses WASM?

Reply:
This does not make much sense, I've re-written it.